unit/ctest_mat.c

```diff
@@ -733,6 +732,8 @@ void test_cu_mat_mm_arrays()
 
   cu_mat_mm_array(cuh, alpha, beta, transa, mat_Acu, transb, array_xcu, array_ycu);
```

Comment:
Is this a public-facing API? I do not think this is correct. We need all methods to be prefixed by the gkyl_ prefix. Also, what is cuh? I thought no handles are to be exposed?